crates/runtime/src/http/v1/iceberg/namespace.rs

```diff
@@ -66,3 +66,15 @@ impl Serialize for Namespace {
         seq.end()
     }
 }
+
+#[derive(Deserialize)]
+pub struct NamespacePath(String);
```

Comment:
This needs a `#[cfg_attr(feature = "openapi", derive(utoipa::ToSchema))]`.

I will update CI to ensure we test with `feature="openapi"`

Reply:
I'll add it in my next PR